components/google-cloud/google_cloud_components/aiplatform/__init__.py

```diff
@@ -13,18 +13,74 @@
 # limitations under the License.
 """Core modules for AI Platform Components."""
 
-from google.cloud import aiplatform as aip
```

Comment:
The import is broken if i'm not mistaken we either need to add a synonym for aiplatfrom or in the code below include the full path to avoid confusion at load time between google-cloud-componets.aiplatfrom versus google.cloud.aiplatfrom .... this makes me thing we may want to follow Ajay's guidance and remove the subfolder in google-cloud-components/aiplatfrom and have everything flat under google-cloud-components.

```
>>> import google_cloud_components
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/google/home/chavoshi/miniconda3/envs/mb/lib/python3.8/site-packages/google_cloud_components/__init__.py", line 15, in <module>
    from google_cloud_components import aiplatform
  File "/usr/local/google/home/chavoshi/miniconda3/envs/mb/lib/python3.8/site-packages/google_cloud_components/aiplatform/__init__.py", line 78, in <module>
    ModelBatchPredictOp = utils.convert_method_to_component(
  File "/usr/local/google/home/chavoshi/miniconda3/envs/mb/lib/python3.8/site-packages/google_cloud_components/aiplatform/utils.py", line 338, in convert_method_to_component
    output_metadata_name, output_metadata_type = map_resource_to_metadata_type(output_type)
TypeError: cannot unpack non-iterable NoneType object
```